pydm/widgets/base.py

```diff
@@ -87,9 +87,14 @@ def refresh_style(widget):
     widgets = [widget]
     widgets.extend(widget.findChildren(QWidget))
```

Comment:
What if `widget` itself was destroyed; will `findChildren` raise?

Reply:
You are correct! It was an oversight.